chapters/ch48.md

```diff
@@ -1,55 +1,55 @@
 
 > 2. Phoneme recognizer: Some linguists believe that there are basic units of sound called "phonemes." For example, the initial "k" sound in "keep" is the same phoneme as the "c" sound in "cake." This system tries to recognize the phonemes in the audio clip.
 
---> _replace THIS LINE by your translation for the above line_
+2. Nhận diện các âm vị: Các nhà ngôn ngữ học tin rằng trong ngôn ngữ có các đơn vị cơ bản gọi là "âm vị." Ví dụ, âm bắt đầu "k" trong từ "keep" thì phát âm giống âm của "c" trong từ "cake". Hệ thống này sẽ cố gắng để nhận diện các âm vị trong các đoạn âm thanh.
 
 > 3. Final recognizer: Take the sequence of recognized phonemes, and try to string them together into an output transcript.
 
---> _replace THIS LINE by your translation for the above line_
+3. Nhận dạng cuối cùng: Dùng các chuỗi âm vị đã được nhận dạng, và cố gắng xâu chuỗi chúng lại với nhau thành một bản ghi thoại đầu ra. 
 
 > In contrast, an end-to-end system might input an audio clip, and try to directly output the transcript:
 
---> _replace THIS LINE by your translation for the above line_
+Mặt khác, một hệ thống đầu-cuối có thể nhận đầu vào là một đoạn hội thoại, và sẽ cố gắng cho ra trực tiếp một bản ghi thoại:
 
 ![img](../imgs/C48_02.png)
 
 > So far, we have only described machine learning "pipelines" that are completely linear: the output is sequentially passed from one staged to the next. Pipelines can be more complex. For example, here is a simple architecture for an autonomous car:
 
---> _replace THIS LINE by your translation for the above line_
+Từ trước tới giờ, chúng ra chỉ mới mô tả các "pipeline" của học máy tuyến tính: đầu ra được truyền tuần tự từ giai đoạn này sang giai đoạn khác. Những Pipeline có thể phức tạp hơn như trong ví dụ về một hệ thống xe tự lái đơn giản sau:
 
 ![img](../imgs/C48_03.png)
 
 > It has three components: One detects other cars using the camera images; one detects pedestrians; then a final component plans a path for our own car that avoids the cars and pedestrians.
 
---> _replace THIS LINE by your translation for the above line_
+Hệ thống bao gồm ba thành phần: Một thành phần giúp phát hiện những xe khác bằng việc sử dụng hình ảnh từ máy quay; Một thành phần khác phát hiện người đi bộ; thành phần cuối cùng hoạch định một lộ trình giúp chiếc xe tránh những chiếc xe khác và người đi bộ.
 
 > Not every component in a pipeline has to be learned. For example, the literature on "robot motion planning" has numerous algorithms for the final path planning step for the car. Many of these algorithms do not involve learning.
 
---> _replace THIS LINE by your translation for the above line_
+Không phải mọi thành phần trong pipeline phải được học tập. Ví dụ, các nghiên cứu về "hoạch định chuyển động của robot" cần rất nhiều thuật toán cho bước hoạch định lộ trình của chiếc xe. Tuy nhiên có nhiều những thuật toán không yêu cầu việc học tập.
```

Comment:
"các nghiên cứu về "hoạch định chuyển động của robot" cần rất nhiều thuật toán" -> "đã có rất nhiều thuật toán cho ..."

Reply:
"Tuy nhiên có nhiều những" -> "Trong đó có nhiều.."